react-example/.env.example

```diff
@@ -1,10 +1,12 @@
-# To make requests from this example app make sure you first create an .env file
-# and add the API key and JWT Secret to it like so (and uncomment the keys):
-# API_KEY=1234
-# JWT_SECRET=1234
+# Optional environment variables for local development. Start by creating a
+# called .env and add these values to it and change them appropriately.
+# Remember to uncomment the variables!
 
-# You can set the URL for the JWT generator here if needed
-# JWT_GENERATOR=http://localhost:5000/generate
+# Only set BASE_URL if developing locally, as it will take precedence over the production api urls.
+# BASE_URL="https://api.iterable.com/api"
```

Comment:
Can you add back the `JWT_GENERATOR` var here?

Reply:
@mprew97 sure, will do that and also handle the other issues you reported on the ticket.

Reply:
Hi @mprew97 all suggested fixes are done, except the image one as NOT able to reproduce. It will be helpful to reproduce if you can send the link of the image used in the screenshot you sent.